packages/dx-react-grid-bootstrap3/src/templates/table-header-cell.jsx

```diff
@@ -66,13 +84,18 @@ export class TableHeaderCell extends React.PureComponent {
             whiteSpace: 'nowrap',
             overflow: 'hidden',
             textOverflow: 'ellipsis',
+            padding: '3px',
           }}
+          onMouseDown={this.handleMouseDown}
+          onBlur={this.handleBlur}
+          onKeyDown={this.handleClick}
         >
           {allowSorting ? (
             <SortingControl
               align={align}
               sortingDirection={sortingDirection}
               columnTitle={columnTitle}
+              handleClick={this.handleClick}
```

Comment:
Cannot find the definition and usage of this property in the `SortingControl` component.